packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts

```diff
@@ -178,6 +174,7 @@ export async function provisionLocalResource(
   }
 
   if (isConfigUnifyEnabled()) {
+    setPostDataForLocal(envInfo!, localSettings);
```

Comment:
Is `envInfo` modified in `configLocalEnvironment`? If so, move this line after calling `configLocalEnvironment`.

Reply:
Or... will `setPostDataForLocal` be deleted later?

Reply:
When I'm done not storing the ```localSettings``` file, ```setPostDataForLocal``` will be removed.